frontend/src/metabase/lib/time.js

```diff
@@ -119,3 +119,29 @@ export function formatFrame(frame) {
       return frame;
   }
 }
+
+export function getRelativeTime(timestamp) {
+  return moment(timestamp).fromNow();
+}
+
+export function getRelativeTimeAbbreviated(timestamp) {
```

Comment:
@daltojohnso I implemented this to get the timestamps to look like `19 m` based on a thread I saw in on a Moment.js GH issue, but let me know if this is hacky or something.